todo_app/trello.py

```diff
@@ -11,17 +11,17 @@ def __init__(self):
         load_dotenv()
         self.key = os.environ.get('TRELLO_KEY')
         self.token = os.environ.get('TRELLO_TOKEN')
-        self.url = os.environ.get('TRELLO_BASE_URL')
+        self.url = 'https://api.trello.com/1/'
         self.board = os.environ.get('TRELLO_BOARD_ID')
-        self.username = os.environ.get('TRELLO_USER')
         self.list_ids = self.get_card_list_ids()
 
-    def make_api_call_and_check_response(self, url, method='get', data=None):
-        complete_url = f"{self.url}" + url + f"?key={self.key}&token={self.token}"
-        resp = eval("requests." + method.lower() + "('" + complete_url + "', data=" + str(data) + ")")
-        if resp.status_code == 200:
-            # print(json.dumps(resp.json(), indent=4))
+    def make_api_call_and_check_response(self, path, method='get', data=None):
+        complete_url = f"{self.url}" + path + f"?key={self.key}&token={self.token}"
```

Comment:
Given that you're using the f-string formatting anyway, we might as well make this all use that, rather than doing some interpolation and some concatenation, i.e.
```
complete_url = f"{self.url}{path}?key={self.key}&token={self.token}"
```